rkt/app_start.go

```diff
@@ -28,7 +28,7 @@ var (
 	cmdAppStart = &cobra.Command{
 		Use:   "start UUID --app=NAME",
 		Short: "Start an app in a pod",
-		Long:  `Start appz!`,
+		Long:  "This allows starting an existing application inside a mutable pod",
```

Comment:
I don't like this "allows" verbiage - it _does_ start an app in a mutable pod, does it not?